### PR TITLE
Fixes for dev-cluster integration

### PR DESCRIPTION
**Pull request by steezeburger**
* installs wget and ca-certificates in our reusable Dockerfile
* increases `DEFAULT_PFD_FEE` from 2_000 to 100_000

Needed for https://github.com/astriaorg/dev-cluster/issues/38

We needed some Dockerfile and astria-sequencer-relayer changes in our images before Retro has been merged, so I made this branch from that point in time.

### CODE REVIEW

File: containerfiles/Dockerfile [hunk 1]
```diff
@@ -94,5 +94,6 @@ FROM debian:bookworm-slim
 ARG TARGETBINARY
 WORKDIR /app/
 EXPOSE 2450
+RUN apt update && apt install -y wget ca-certificates
```
Comment (SuperFluffy) [inline containerfiles/Dockerfile:97]: Let's chain `&& apt-get clean && rm -rf /var/lib/apt/lists/*`

File: crates/astria-sequencer-relayer/src/data_availability.rs [hunk 2]
```diff
@@ -46,7 +46,7 @@ use crate::{
 };
 
 pub const DEFAULT_PFD_GAS_LIMIT: u64 = 1_000_000;
-const DEFAULT_PFD_FEE: u128 = 2_000;
+const DEFAULT_PFD_FEE: u128 = 100_000;
```
Comment (SuperFluffy) [inline crates/astria-sequencer-relayer/src/data_availability.rs:49]: Please split this out into its own PR because these changes are unrelated (even if small).

Comment (steezeburger) [reply crates/astria-sequencer-relayer/src/data_availability.rs:49]: I needed this all in one place so that I could cut a tag and create updated and working images.

I wouldn't have included this here otherwise.

Comment (SuperFluffy) [reply crates/astria-sequencer-relayer/src/data_availability.rs:49]: Make one branch A with the docker change, then fork a new branch B from A with the gas change.

Open 2 PRs, and for branch B set GitHub to merge into (target) A.

You can now create images on PR B.

Once PR A is merged GitHub will automatically update PR B to merge to main.

Comment (steezeburger) [reply crates/astria-sequencer-relayer/src/data_availability.rs:49]: Wouldn't the final PR get squash merged and ultimately all look the same in main at the end? I'm not sure it's worth all of this effort.

Comment (SuperFluffy) [reply crates/astria-sequencer-relayer/src/data_availability.rs:49]: No, you have 2 PRs that will result in separate commits, so nothing gets squashed.


Re effort: if you had branched off of your original PR this would be less work.
 
You'll have more work writing the commit message explaining why these changes are necessary. :-)

Comment (steezeburger) [reply crates/astria-sequencer-relayer/src/data_availability.rs:49]: My original PR has main changes.

Comment (steezeburger) [reply crates/astria-sequencer-relayer/src/data_availability.rs:49]: For posterity: discussed in Slack and okay with solution to do all the context around this work.

**Review by joroshiba — APPROVED** [review]
